src/SeqViz/handlers/selection.jsx

```diff
@@ -63,15 +63,19 @@ const withSelectionHandler = WrappedComp =>
       document.removeEventListener("mouseup", this.stopDrag);
     };
 
-    componentDidUpdate = prevProps => {
-      if (!isEqual(this.props.selectedRange, prevProps.selectedRange)) {
-        if (!!this.props.selectedRange) {
+    componentDidUpdate = () => {
+      if (this.props.selectedRange === null) {
```

Comment:
We can apply destructuring on selectedRange and selection, since are being used in a lot of places **const { selectedRange, selection } = this.props;**